### PR TITLE
Update filepaths

### PR DESCRIPTION
**Pull request by JoramD**
**When merged this pull request will:**
- Update file-paths for the new TAC_Resources mod.

Should only be merged after TAC_Resources is released/used
https://github.com/Theseus-Aegis/Mods/pull/363

### CODE REVIEW

**Comment by emythiel** [comment]
Good idea to check the scripts as well.
For example, this one:
https://github.com/Theseus-Aegis/Olympus/blob/master/functions/fn_setBillboard.sqf#L20

Though think it would need some changes to the script_component as well, since it uses the macro stuff.
@jonpas

**Comment by jonpas** [comment]
Yeah, this https://github.com/Theseus-Aegis/Olympus/blob/042eccd7c765a1bd26a41850efa39c8350598064/script_component.hpp#L17-L18

I would just create 2 new macros in same format, pointing to `\x\tacr\addons`, and would be used as `QPATHTOTACRF(images\bla.paa)`. Or any shorter name for that macro. :P

File: script_component.hpp [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 #include "\x\cba\addons\main\script_macros_mission.hpp"
 
-#define PATHTOTACF(var1,var2) PATHTOF_SYS(\x\tac\addons,var1,var2)
-#define QPATHTOTACF(var1,var2) QUOTE(PATHTOTACF(var1,var2))
+#define PATHTOTACR(var1,var2) PATHTOF_SYS(\x\tac\addons,var1,var2)
```
Comment (jonpas) [inline script_component.hpp]: Nothing important, just want to say `F` at the end means "file", there is none. (I don't care if there is, depends if you do.)

**Comment by JoramD** [comment]
Whoops sorry about that, should be all fixed now.

File: script_component.hpp (hunk 1 above)
Comment (JoramD) [reply script_component.hpp]: I was wondering what that was for, will change it.

**Comment by jonpas** [comment]
Well you have to change the actual usage too! 😆

**Comment by jonpas** [comment]
TAC Resources is released and used. @SyMP2005 you can review/test/merge this.

**Pull request merged**